<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#titleLabel {
  background: white;
  border: none;
  border - bottom: 1px solid black;
  padding: 5px;
  font: 16pt &quot;黑体&quot;;
  color: rgba(255, 255, 255);
  background - color: rgba(0, 170, 127, 255);
}</string>
    </property>
    <property name="text">
     <string>袁伟珉的聊天室</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginpage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>weimin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>登陆聊天室</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chatpage">
     <widget class="QListWidget" name="userlistWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>61</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="roomtextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>236</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="saylineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sayButton">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
